src/Patterns/BlockPattern.php

```diff
@@ -41,6 +41,7 @@ public static function get_classname(): string
     public static function get_list(): array
     {
         return [
+            Action::class,
```

Comment:
I think for this one we shouldn't follow the alphabetical order, because we want the "Blank Page" one to show first